boosters/node-beginner/tests/config/teardown.js

```diff
@@ -1,6 +1,6 @@
 const db = require("../../models");
 
 module.exports = async () => {
-  await Promise.all([db.Users.truncate(), db.Repos.truncate()]);
+  await db.Users.truncate({ cascade: true });
```

Comment:
Does the `Repos` table need to be truncated as well?

Reply:
Responded on Slack, but truncating the Users with `cascade:t true` is fine since all  repos records need to have a userId

Reply:
Alright. One thing though is that in the `Repos` table, the `userId` foreign constraint in nullable. Not a big issue with our test cases, but if a repo record somehow gets updated to remove that `userId` reference, those records wouldn't get cascaded on a DELETE operation (could be a valid business case).